### ISSUE TITLE
FreeCAD ?

### ISSUE THREAD

**Issue by luzpaz** [UWP-Open-Source-Community/Documentation#5]
FOSS 3D Parametric CAD 
https://www.freecadweb.org/

**Comment by duble0** [comment]
Yes, we planned to convert it, but is a very huge software, so when we have a little bit of time we'll start to convert it...thank you for your suggest! ;-)

**Comment by luzpaz** [comment]
Fantastic. Can you ping me when that happens? 
CC @sgrogan

**Comment by duble0** [comment]
yes, sure!
